specs/catalog-schema-registry.spec/src/main/scripts/io/aklivity/zilla/specs/catalog/schema/registry/schema/schema.registry.schema.patch.json

```diff
@@ -59,6 +64,51 @@
                     "title": "Max Age",
                     "type": "integer",
                     "default": 300
+                },
+                "keys":
+                {
+                    "title": "Vault Keys",
+                    "type": "array",
+                    "items":
+                    {
+                        "type": "string"
+                    }
+                },
+                "trust":
+                {
+                    "title": "Vault Certificates",
+                    "type": "array",
+                    "items":
+                    {
+                        "type": "string"
+                    }
+                },
+                "trustcacerts":
+                {
+                    "title": "Trust CA Certificates",
+                    "type": "boolean"
+                },
```

Comment:
These feel like they need a grouping, as they are lower level compared to the other options.

Perhaps `encryption`, `security`, `tls`, `https`?